src/bitcoind.cpp

```diff
@@ -43,10 +43,6 @@ static bool fDaemon;
 
 void WaitForShutdown(boost::thread_group* threadGroup)
 {
-    // This is the main process thread, but after this point, all we do here is wait for
-    // shutdown, and then shut down the node.
```

Comment:
I guess the comment is still correct. Let me know if you want to retain it (but I think it's fairly obvious from the name of the function).